extensions/sd_api_pictures/script.py

```diff
@@ -66,19 +65,29 @@ def get_SD_pictures(description):
 
     global params, pic_id
 
+    if params['manage_VRAM']:
+        shared.gradio['unload_model_fn']()
+        response = requests.post(url=f'http://{params["address"]}/sdapi/v1/reload-checkpoint', json='')
+        response.raise_for_status()
+        r = response.json()
+        del response
+        del r
+
     payload = {
         "prompt": params['prompt_prefix'] + description,
         "seed": -1,
```

Comment:
Would be nice to have seed, sampler_name, steps and cfg_scale configurable as well.

Reply:
I don't know why you'd want to manually specify a seed when an AI is randomly generating the prompt?

Reply:
some times same seed provides similar images on slightly different promts, it helps to get more consistent characters/visuals (or I am just seeing things)